### ISSUE TITLE
Include statement

### ISSUE THREAD

**Issue by CamelCaseCam** [Koeng101/parts#4]
Have you considered adding an include statement field to divide content from toolkits between multiple organisms? Otherwise, these files could get pretty big. 

I'm thinking something simple like `includes: ["x", "y", "z"]`

**Comment by Koeng101** [comment]
Could you give a specific example based off of the [current schema](https://github.com/Koeng101/parts/blob/main/collections/bsub.yaml)?

**Comment by CamelCaseCam** [comment]
yup - if we look at the pichia [file](https://github.com/CamelCaseCam/parts/blob/main/collections/pichia.yaml) in my fork, it reads
```
pAOX1:
    ...
pGAP:
    ...
```
and so on in the promoter section. I would like to have a file called `plasmids` at `collections/pichia` and instead put `includes: ["pichia/plasmids"]` to keep things more organized.

**Comment by Koeng101** [comment]
Oh, so lemme get this straight:

`collections.yaml`, rather than just having something like `file: collections/pichia.yaml`, you have `includes: ["pichia/plasmids/promoter.yaml"] `, right?

**Comment by CamelCaseCam** [comment]
Not exactly. You'd still have `file: collections/pichia.yaml`, but in pichia.yaml I could put `includes: ["pichia/plasmids/promoter.yaml"]`

**Comment by CamelCaseCam** [comment]
Fair enough - that makes sense!
